Tools/CMake/AMReX_Options.cmake

```diff
@@ -90,15 +90,33 @@ endif ()
 print_option( DIM )
 
 #
-# Programming models  ========================================================
+# Parallel backends    ========================================================
 #
-option( ENABLE_MPI  "Enable MPI"  ON)
+
+# For the time being ENABLE_DPCPP is defined before project() is called
+# Check whether the C++ compiler is dpcpp
+print_option(ENABLE_DPCPP)
+if (ENABLE_DPCPP AND (NOT (CMAKE_CXX_COMPILER MATCHES "dpcpp") ) )
+   message(FATAL_ERROR "\nENABLE_DPCPP=${ENABLE_DPCPP} but CXX compiler is not dpcpp\n")
+endif ()
+
+cmake_dependent_option( ENABLE_DPCPP_AOT  "Enable DPCPP AOT"  OFF
+   "ENABLE_DPCPP" OFF)
+print_option( ENABLE_DPCPP_AOT )
+
+cmake_dependent_option( ENABLE_DPCPP_SPLIT_KERNEL "Enable DPCPP kernel splitting"  ON
+   "ENABLE_DPCPP" OFF)
+print_option(  ENABLE_DPCPP_SPLIT_KERNEL )
+
+cmake_dependent_option( ENABLE_MPI  "Enable MPI"  ON
```

Comment:
Does DPC++ not work with MPI?

Reply:
In the GNUMake MPI is turned off if DPC++ is on, so I guess it does not.

Reply:
@WeiqunZhang can you please clarify this aspect? Do MPI and DPC++ work together already or is there another reason we miss?

Reply:
We have never tested it and don't really have a place to test it.  So I simply disable it for now in GNU make.  However, there is no reason it won't work.